contracts/common/ConstantPriceFeed.sol

```diff
@@ -7,7 +7,7 @@ import { IPriceOracle } from "../Oracles/Interfaces/IPriceOracle.sol";
 import { Lock } from "./Lock.sol";
 
 /// @dev Price oracle to be used for peg stability module to mint R.
-/// Returns constant price of 1 USD per token with 0 deviation.
+/// Returns constant price of 2 USD per token with 0 deviation.
```

Comment:
Should be 1 USD

Reply:
https://github.com/raft-fi/contracts/pull/458

Reply:
lgtm